### PR TITLE
Load dashboard templates as frontend cards

### PR DESCRIPTION
**Pull request by tiagojsag**
Missing (for other PRs)
- create the actual links to the dashboards
- links to indicators

### CODE REVIEW

**Comment by tiagojsag** [comment]
ready for review

File: app/admin/api/v3/dashboard_template.rb [hunk 1]
```diff
@@ -16,6 +18,7 @@ def clear_cache
     inputs do
       input :title, as: :string, required: true
       input :description, required: true
+      input :category
```
Comment (agnessa) [inline app/admin/api/v3/dashboard_template.rb]: is this free text, or possible to have an enumeration with possible values in a dropdown?

Comment (tiagojsag) [reply app/admin/api/v3/dashboard_template.rb]: AFAIK there's no definition on that, so I'm leaving it open (and optional) for now.

Comment (agnessa) [reply app/admin/api/v3/dashboard_template.rb]: 👌

**Pull request merged**